Detectors/TRD/simulation/include/TRDSimulation/TrapConfig.h

```diff
@@ -34,7 +35,7 @@ namespace trd
 class TrapConfig
 {
  public:
-  TrapConfig(std::string configname);
+  TrapConfig(std::string configname = "default");
```

Comment:
What is the "default" config? It would be better to provide the actual name.

Reply:
Its a carry over from the old way of initialising the trapconfig via the trapconfighandler which is an open question whether we need trapconfighandler. The default is specified on the command line to the trapsimulator dpl device. Removing.

Reply:
trapconfighandler code is still there, just not used anymore in device.